seed/challenges/02-javascript-algorithms-and-data-structures/regular-expressions.json

```diff
@@ -305,7 +305,7 @@
         "Using the hyphen (<code>-</code>) to match a range of characters is not limited to letters. It also works to match a range of numbers.",
         "For example, <code>/[0-5]/</code> matches any number between <code>0</code> and <code>5</code>, including the <code>0</code> and <code>5</code>.",
         "Also, it is possible to combine a range of letters and numbers in a single character set.",
-        "<blockquote>let jennyStr = \"Jenny8675309\";<br>let myRegex = /[a-z0-9]/ig;<br>jennyStr.match(myRegex); // matches all letters and numbers in jennyStr</blockquote>",
+        "<blockquote>let jennyStr = \"Jenny8675309\";<br>let myRegex = /[a-z0-9]/ig;<br>jennyStr.match(myRegex);<br>// matches all letters and numbers in jennyStr</blockquote>",
```

Comment:
@cdrainxv What do you think about moving the comment to the line before the regex?